Tribler/community/tunnel/crypto/tunnelcrypto.py

```diff
@@ -43,13 +43,17 @@ def verify_and_generate_shared_secret(self, dh_secret, dh_received, auth, B):
         return shared_secret
 
     def generate_session_keys(self, shared_secret):
+        # TODO hkdf???
         hkdf = HKDFExpand(algorithm=hashes.SHA256(), backend=default_backend(), length=40, info="key_generation")
         key = hkdf.derive(shared_secret)
 
+        # TODO what is this
```

Comment:
We've asked the HKDF for a string of length 40. This gives us:
* two 128 bit keys
* two 32 bit salts